tests.py

```diff
@@ -0,0 +1,46 @@
+__copyright__ = "Copyright 2017 Birkbeck, University of London"
+__author__ = "Martin Paul Eve, Andy Byers & Mauro Sanchez"
+__license__ = "AGPL v3"
+__maintainer__ = "Birkbeck Centre for Technology and Publishing"
+
+from plugins.typesetting import plugin_settings
+from utils.testing import helpers
+
+
+class TestTypesetting(TestCase):
+    # Tests for editor role checks
+
+    @classmethod
+    def setUpTestData(self):
```

Comment:
The setup method looks ok, but there aren't any tests AFAIC

Reply:
Done.